docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml

```diff
@@ -0,0 +1,376 @@
+      sno-ran-du-policy: "v4"
+  remediationAction: enforce
+  severity: low
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 5m
+    noncompliant: 10s
+  orderPolicies: true
+policies:
+# REDUCE FOOTPRINT
+- name: v4-footprint-policy
+  manifests:
+    # Do not add retention field for prometheusK8s when observability is enabled.
+    # It will be overridden by OBS.
+    - path: source-crs/ReduceMonitoringFootprint.yaml
+      patches:
+      - data:
+          config.yaml: |
+            alertmanagerMain:
+              enabled: false
+            telemeterClient:
+              enabled: false
+# CATALOG SOURCE
+- name: v4-catalog-source-policy
+  manifests:
+    - path: source-crs/DefaultCatsrc.yaml
+      patches:
+      - metadata:
+          name: redhat-operators
+        spec:
+          displayName: redhat-operators
+          image: registry.redhat.io/redhat/redhat-operator-index:v4.Y
+    - path: source-crs/DefaultCatsrc.yaml
+      patches:
+      - metadata:
+          name: certified-operators
+        spec:
+          displayName: certified-operators
+          image: registry.redhat.io/redhat/certified-operator-index:v4.Y
+# SUBSCRIPTIONS
+- name: v4-subscriptions-policy
+  manifests:
+    # Cluster Logging operator
+    - path: source-crs/ClusterLogNS.yaml
+    - path: source-crs/ClusterLogOperGroup.yaml
+    - path: source-crs/ClusterLogSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/ClusterLogOperatorStatus.yaml
+    - path: source-crs/ClusterLogServiceAccount.yaml
+    - path: source-crs/ClusterLogServiceAccountAuditBinding.yaml
+    - path: source-crs/ClusterLogServiceAccountInfrastructureBinding.yaml
+    # PTP operator
+    - path: source-crs/PtpSubscriptionNS.yaml
+    - path: source-crs/PtpSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub or (index $configMap.data "install-plan-approval") "Manual" hub}}'
+    - path: source-crs/PtpSubscriptionOperGroup.yaml
+    - path: source-crs/PtpOperatorStatus.yaml
+    # SRIOV operator
+    - path: source-crs/SriovSubscriptionNS.yaml
+    - path: source-crs/SriovSubscriptionOperGroup.yaml
+    - path: source-crs/SriovSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/SriovOperatorStatus.yaml
+    # SRIOV Accelerator
```

Comment:
Optional component. If the deployment needs this only under some conditions is the right solution to create a second policytemplate without it?

Reply:
For this usecase it is going to be required, need to include in our example